tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -43,6 +44,134 @@ public function test_get_site_fields() {
 		);
 	}
 
+	public function test_fetch_site_fields() {
```

Comment:
@benbowler let's also add a case when proxy returns an error to make sure that `fetch_site_fields` returns an appropriate WP_Error instance.